test/cmdlineTests.sh

```diff
@@ -113,14 +109,6 @@ do
 done
 )
 
-printTask "Compiling all files in std and examples..."
-
-for f in "$REPO_ROOT"/std/*.sol
-do
-    echo "$f"
-    compileWithoutWarning "$f"
```

Comment:
It seems that we were running the compilation twice :)

Reply:
The first one actually checked that they produce bytecode, i.e. that the contracts are not abstract and that the bytecode generator works.